.buildkite/react-native-pipeline.yml

```diff
@@ -110,3 +110,109 @@ steps:
       APP_LOCATION: "build/rn0.63.ipa"
     concurrency: 10
     concurrency_group: 'browserstack-app'
+
+  - label: ':android: Build react-navigation 0.60 apk'
+    env:
+      REACT_NATIVE_VERSION: "rn0.60"
+      JS_SOURCE_DIR: "react_navigation_js"
+      ARTEFACT_NAME: "r_navigation_0.60"
+    plugins:
+      - docker-compose#v3.3.0:
+          run: react-native-android-builder
+    artifact_paths:
+      - build/r_navigation_0.60.apk
+
+  - label: ':ios: Build react-navigation 0.60 ipa'
+    agents:
+      queue: "opensource-mac-rn"
+    env:
+      REACT_NATIVE_VERSION: rn0.60
+      JS_SOURCE_DIR: "react_navigation_js"
+      ARTEFACT_NAME: "r_navigation_0.60"
+    artifact_paths: build/r_navigation_0.60.ipa
+    commands:
+      - npm run test:build-react-native-ios
+
+  - label: ':android: Build react-navigation 0.63 apk'
+    env:
+      REACT_NATIVE_VERSION: "rn0.63"
+      JS_SOURCE_DIR: "react_navigation_js"
+      ARTEFACT_NAME: "r_navigation_0.63"
+    plugins:
+      - docker-compose#v3.3.0:
+          run: react-native-android-builder
+    artifact_paths:
+      - build/r_navigation_0.63.apk
+
+  - label: ':ios: Build react-navigation 0.63 ipa'
+    agents:
+      queue: "opensource-mac-rn"
+    env:
+      REACT_NATIVE_VERSION: rn0.63
+      JS_SOURCE_DIR: "react_navigation_js"
+      ARTEFACT_NAME: "r_navigation_0.63"
+    artifact_paths: build/r_navigation_0.63.ipa
+    commands:
+      - npm run test:build-react-native-ios
+
+  - wait
+
+  - label: ':android: React-navigation 0.60 Android 9'
+    plugins:
+      artifacts#v1.2.0:
+        download: "build/r_navigation_0.60.apk"
+      docker-compose#v3.3.0:
+        run: react-native-maze-runner
+        use-aliases: true
+        command: ["features/navigation.feature"]
+    env:
+      DEVICE_TYPE: "ANDROID_9"
+      APP_LOCATION: "build/r_navigation_0.60.apk"
+      RUN_NAVIGATION_SCENARIOS: "true"
+    concurrency: 10
+    concurrency_group: 'browserstack-app'
+
+  - label: ':ios: React-navigation 0.60 iOS 12'
```

Comment:
Perhaps we should target iOS 13 or even 14 for the navigation tests, so that we have a greater diversity covered by the pipeline in some way in the short term.  Longer term of course we'll probably seek to run all combinations before a release and smoke them on each push.